context/myuuid.go

```diff
@@ -11,13 +11,14 @@ import (
 )
 
 var (
-	myUUIDFile = path.Join(config.GetString("localstatedir"), "uuid")
+	myUUIDFile string
 )
 
 // InitMyUUID initializes MyUUID by reading the `<config.LocalStateDir>/uuid` file.
 // If the file is not present it generates a new UUID and saves it to the file.
 // If the file is not accessible. initMyUUID panics.
 func InitMyUUID() uuid.UUID {
+	myUUIDFile = path.Join(config.GetString("localstatedir"), "uuid")
```

Comment:
I guess just this should be enough to fix the issue.

Global variables get initialized during package initialization when a Go binary is loaded. This happens before any functions are called. The previous config package we had initialized LocalStateDir during package initialization, as the `flag` package parses the command line on init (be default).

With the change to Viper, the config variables are now only ready for use after `initConfig()` is called. `InitMyUUID` gets called after `initConfig()` in `main()`. So just having this one change should be enough to the UUID file working properly.